lxd/db/images.go

```diff
@@ -785,6 +809,18 @@ func (c *Cluster) ImageInsert(project, fp string, fname string, sz int64, public
 		arch = 0
 	}
 
+	imageType := -1
```

Comment:
@stgraber if we change `instance.New()` to return a more generic "invalid type" style message, you could use that here (and would require us to add a virtual-machine entry which I will need to do anyway).